deploy/helm/charts/values.yaml

```diff
@@ -132,6 +132,10 @@ lvmPlugin:
     containerRuntime: containerd
     readIopsPerGB: ""
     writeIopsPerGB: ""
+  # The TCP network address where the prometheus metrics endpoint will listen.
+  # The default is empty string, which means metrics endpoint is disabled.
+  # If not set, service will not be created to expose this endpoint to serviceMonitor.
+  metricsListenAddress: 9080
```

Comment:
@akhilerm @iyashu Should we assign some default value to metricsListenAddress or should it be kept empty?
For now, I have assigned `9080`.

Reply:
Can we keep the behavior similar to other OpenEBS engines - cstor, jiva? If we are enabling metrics by default in cstor/jiva, I think we should do the same in lvm-localpv as well.

Reply:
Enabled by default. 
metricsListenAddress: 9500